src/main/java/blackJack/domain/card/Cards.java

```diff
@@ -0,0 +1,54 @@
+package blackJack.domain.card;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Cards {
+    private static final int BLACK_JACK_CARD_COUNT = 2;
+    private static final int BLACK_JACK = 21;
+    private static final int OTHER_SCORE_OF_ACE_DENOMINATION = 11;
+
+    private final Set<Card> cards;
+
+    public Cards() {
+        cards = new HashSet<>();
+    }
+
+    public void receiveCard(Card card) {
+        cards.add(card);
+    }
+
+    public boolean isBlackJack() {
+        return cards.size() == BLACK_JACK_CARD_COUNT && calculateFinalScore() == BLACK_JACK;
+    }
+
+    public boolean isBust() {
+        return calculateFinalScore() > BLACK_JACK;
+    }
+
+    public int calculateFinalScore() {
+        final int score = calculateScore();
+        if (hasAce() && score + OTHER_SCORE_OF_ACE_DENOMINATION - Denomination.ACE.getScore() <= BLACK_JACK) {
+            return score + OTHER_SCORE_OF_ACE_DENOMINATION - Denomination.ACE.getScore();
+        }
+        return score;
+    }
```

Comment:
아래와 같은 형태로 리팩토링 해보면 어떨까요?
조금 더 읽기 좋은 코드가 될 것 같아요!
```suggestion
        if (hasAce()) {
          return 메서드(score);
        }
        return score;
    }
```

Reply:
점수를 더 받을 수 있는지 확인하는 조건문에 hasAce()와 함께 버스트가 되는 점수인지 확인하는 메서드는 함께 있는게 좋다고 생각해서 조금은 다른 방향으로 아래처럼 수정하였습니다!

```
if (hasAce() && checkValidationAceBonusScore(score)) {
    return score + ACE_BONUS_SCORE;
}
```